pkg/netpol/connlist/conns_formatter_txt.go

```diff
@@ -22,7 +22,7 @@ type formatText struct {
 func (t *formatText) writeOutput(conns []Peer2PeerConnection, exposureConns []ExposedPeer, exposureFlag bool) (string, error) {
 	res := t.writeConnlistOutput(conns, exposureFlag)
 	if !exposureFlag {
-		return res, nil
+		return res + newLineChar, nil
```

Comment:
better place the addition of the `newLineChar`  where `Join` is called, within `writeConnlistOutput`, as this is directly related to the result from `join`, where it does not add newline for that result.

Reply:
done